### DISCUSSION TITLE
acr_values

### DISCUSSION THREAD

**Discussion by agustingabiola** [panva/openid-client#319] [Q&A] [answered]
Hello @panva great library. One question, is there a way to avoid encoding the acr_values? I have a particular use case where the server doesn't want the + encoded and I have bypassed this by generating the url and appending the values but wanted to know if there is a flag to turn off encoding in certain parameters.

Thank you in advance. Keep up the good work :)

**Comment by panva** [comment, marked as answer]
There's no way to skip url encoding of query string parameters.

**Reply by agustingabiola** [reply]
Thanks for the swift response and have a good day!
